index.js

```diff
@@ -7,13 +7,17 @@ var getGitInfo = require('git-repo-info');
 
 module.exports = function version(shaLength, root) {
   var projectPath = root || process.cwd();
-
+  var packageVersion  = require(path.join(projectPath, 'package.json')).version;
   var info = getGitInfo(projectPath);
+
   if (info.tag) {
-    return info.tag;
+    if (packageVersion && info.tag.includes(packageVersion)) {
```

Comment:
I think that we should, since this package is used by all ember-cli apps, and ember-cli supports node 0.10.